capstone-backend/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -29,6 +37,30 @@
 @WebServlet("/api/clubs")
 public class ClubServlet extends HttpServlet {
 
+  private JsonObject createRequestBodyJson(HttpServletRequest request) {
+    JsonObject jsonObject = new JsonObject();
+    StringBuffer jb = new StringBuffer();
+    String line = null;
+    try {
+      BufferedReader reader = new BufferedReader(new InputStreamReader((request.getInputStream())));
+      while ((line = reader.readLine()) != null) {
+       jb.append(line);
+      }
+    } catch (Exception e) { 
+      System.err.println("Error: " + e);
+    }
+    try {
+      Gson gson = new Gson();
+      JsonParser parser = new JsonParser();
```

Comment:
According to Gitpod, ```JsonParser``` is deprecated. May lead to issues but it I think is fine.

Reply:
fixed!